src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -0,0 +1,36 @@
+package juja.sqlcmd;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DatabaseManagerTest {
+    private DatabaseManager databaseManager;
+
+    @Before
+    public void init() {
+        databaseManager = new DatabaseManager();
+    }
+
+    @Test
+    public void connectWhenValidDataReturnsTrue() {
+        assertTrue(databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
+    }
```

Comment:
You could name it "connectWhenValidAllParametersReturnsTrue"

Reply:
It sounds realy better... Thanks!